app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsActivity.kt

```diff
@@ -21,6 +28,23 @@ fun Activity.startToolDetailsActivity(toolCode: String) {
     )
 }
 
+fun Activity.launchTrainingTips(
+    tool: Tool?,
+    type: Tool.Type,
+    translation: Translation,
+    isDiscovered: Boolean = false,
+    force: Boolean
+) {
+    val code = tool?.code ?: return
+    val locale = translation?.languageCode
+
+    if (force || isDiscovered) openToolActivity(code, type, locale, showTips = true)
+    else startTutorialActivity(PageSet.TIPS, Bundle().apply {
+        putString(TutorialActivity.FORMAT_ARG_TOOL_NAME, translation.getName(tool).toString())
```

Comment:
I also don't think we need to send this string to the tutorial anymore. I'm pretty sure we removed the tool name from the tutorial